check_plugins_viewer_mandatory_fields.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+FIELDS=("title" "publisher" "category" "icon" "description" "repository" "firstPublicationDate" "latestUpdateDate")
```

Comment:
What about URL, attributes fields, should we check them too?

Reply:
I only check in this script for plugin viewer specific fields, that were given in the originial issue description